lib/retry.test.ts

```diff
@@ -60,7 +66,7 @@ describe("createDynamicImportWithRetry bust the cache of a module using the curr
 
     const /* ignored */ _promise = dynamicImportWithRetry(
         originalImport as any
-      );
+      ).catch(console.log);
```

Comment:
Pretty sure the `log` function cannot be called without being attached the the `console`, so this should not work. I will instead use the [debug based](https://www.npmjs.com/package/debug) logger that can be turned on/off with an environment variable.

```suggestion
      ).catch(logger);
```